test/sdk-benchmarks/src/main/java/software/amazon/awssdk/benchmark/dynamodb/AbstractItemFactory.java

```diff
@@ -0,0 +1,144 @@
+package software.amazon.awssdk.benchmark.dynamodb;
```

Comment:
It seems a bit odd to me the groupId is `com.dongieagnir` but the package is under `software.amazon.awssdk` and do we need to put copyright to all the files in this module?

Reply:
Oops moved the package but not the group. I'll fix.